src/views/Pipeline.vue

```diff
@@ -821,5 +821,14 @@ ion-chip {
   ion-toolbar > div > ion-chip:first-child {
     margin-left: var(--spacer-3xl);
   }
+  /* Added CSS styles to the <aside> element to occupy 80% of the viewport height, enable vertical scrolling for overflowing content,
+  hide the scrollbar, and add bottom padding for spacing */
+  aside {
+    height: 80vh;
```

Comment:
we can't use a fixed number here. Need to take approach similar to my MR where I used 100% height of parent container.